ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3257,4 +3257,8 @@ public interface EjbLogger extends BasicLogger {
     @LogMessage(level = WARN)
     @Message(id = 531, value = "No client mappings registry provider found for %s; using legacy provider based on static configuration")
     void legacyClientMappingsRegistryProviderInUse(String name);
+
+    @LogMessage(level = WARN)
+    @Message(id = 532, value = "Database detected from configuration is: '%s' , If this is incorrect, please specify the correct database.")
```

Comment:
This needs fixing: there is an extra space before the colon and "If" is incorrectly capitalized.

Reply:
Before the colon can't able to see any extra space, I guess it will be the comma. Changed it accordingly.

Reply:
Actually it should either be a semi-colon and small i or a full-stop/period with a capital I.

Reply:
Thanks @bstansberry changed it accordingly.